compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.td

```diff
@@ -1568,10 +1611,19 @@ def FLOW_DispatchWorkloadOrdinalOp :
   let hasFolder = 1;
 }
 
+} // OpGroupTensorOps
+
 //===----------------------------------------------------------------------===//
 // Collective communication ops
 //===----------------------------------------------------------------------===//
 
+def OpGroupCollectiveCommunicationOps : OpDocGroup {
```

Comment:
Alphabetic sorting puts this group at the top of the page when it's at the bottom in the source file. If we care, we could change the default sort mode or add a flag that this file can set.

Reply:
Yeah without groupings its alphabetical, so when combined I tried to retain that.